.editorconfig

```diff
@@ -259,52 +262,79 @@ dotnet_diagnostic.IDE2000.severity = warning
 [{eng/tools/**.cs,**/{test,samples,perf}/**.cs}]
 # CA1018: Mark attributes with AttributeUsageAttribute
 dotnet_diagnostic.CA1018.severity = suggestion
+
```

Comment:
Could you undo this? The spaces were meant to make comments or description much clearer, but these are all suppressions for tests which don't need that.